### PR TITLE
SAK-33424 Assignments submission notifications

### PR DESCRIPTION
**Pull request by ern**
- Added EmailUtil organizing email methods to that class

### CODE REVIEW

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+        else
+            message.append(htmlContentReleaseResubmission(s));
+        message.append(htmlContentAttachments(s));
+        message.append(htmlEnd());
+        message.append(TERMINATION_LINE);
+        return message.toString();
+    }
+
+    public String plainTextHeaders() {
+        return "Content-Type: text/plain\n\n";
```
Comment (ottenhoff) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java]: are you sure you want double line break?

Comment (ern) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java]: Yeah thats how it was previously, but I agree not sure why it was like that.

Comment (ottenhoff) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java]: yeah, i remember seeing issues with PHP and multiple line breaks telling the email client that the body was starting .... so if the body is indeed next, then this seems fine

https://stackoverflow.com/questions/30887610/error-with-php-mail-multiple-or-malformed-newlines-found-in-additional-header

Comment (ottenhoff) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java]: seems like the contents follow so this is indeed correct

**Pull request merged**